src/app/common/TrayView.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <KSlideout
+    close-button-alignment="end"
+    :has-overlay="false"
+    is-visible
+    :title="props.title"
+    data-testid="tray"
+    @close="emit('close')"
+  >
+    <template
+      v-if="$slots.icon"
+      #before-title
+    >
+      <slot name="icon" />
+    </template>
+
+    <slot />
+  </KSlideout>
+</template>
+
+<script lang="ts" setup>
+const props = defineProps<{
+  title: string
+}>()
```

Comment:
Are we ever going to need to set a title that gets put inside a p tag inside the tray? Actually its kinda hard to see in this PR how this component would be used, I'd assumed we were gonna but a h* in the default slot. Actually, didn't we say you were gonna add a "usage commit" on the end here that we could just remove once we were happy?

Reply:
I said very explicitly that I would keep https://github.com/kumahq/kuma-gui/pull/1611 open for that purpose and is linked from the PR (very first line) and from the ticket in the board.

Reply:
Oh gotcha, sorry misunderstood.

That PR (1611) still has the old custom tray? Can we get it updated to use the stuff form here then so I can see what usage looks like altogether?

Reply:
> Are we ever going to need to set a title that gets put inside a p tag inside the tray?

Yes. I plan to change this in KSlideout put for now we have no control over the container element for `props.title`.

Reply:
I did update both PRs now